admin_menu/templatetags/custom_admin_menu.py

```diff
@@ -7,9 +7,11 @@
 from django.utils.text import capfirst
 from django.utils.translation import ugettext_lazy as _
 from django.apps import apps
+from django import VERSION
 
 register = template.Library()
 
+SUPPORTS_VIEW_PERM = True if (VERSION[0] == 2 and VERSION[1] > 0) or (VERSION[0] > 2) else False
```

Comment:
```suggestion
SUPPORTS_VIEW_PERM = (VERSION[0] == 2 and VERSION[1] > 0) or (VERSION[0] > 2)
```

If you’re checking a boolean to return a boolean, you can just return that boolean!